tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js

```diff
@@ -23,32 +22,110 @@ class LifecycleUtility extends ReplicationUtility {
         return this;
     }
 
+    setSourceLocation(sourceLocation) {
+        this.sourceLocation = sourceLocation;
+        return this;
+    }
+
     setDestinationLocation(destinationLocation) {
         this.destinationLocation = destinationLocation;
         return this;
     }
 
+    setLocationType(locType) {
+        this.locationType = locType;
+        return this;
+    }
+
+    createBucket(cb) {
+        if (this.sourceLocation) {
+            this.s3.createBucket({
```

Comment:
Minor: if we check `this.sourceLocation` to be 'Zenko', then we can create the bucket and return. That way we can avoid the else block.